src/type.js

```diff
@@ -376,6 +385,19 @@ function typeCharacter(
   }
 }
 
+function fireChangeForInputTimeIfValid(
+  currentElement,
+  prevValue,
+  timeNewEntry,
+) {
+  if (
+    isValidInputTimevalue(currentElement(), timeNewEntry) &&
+    prevValue !== timeNewEntry
+  ) {
+    fireEvent.change(currentElement(), {target: {value: timeNewEntry}})
+  }
+}
+
```

Comment:
I think we don't need this to be a function. Could you inline it please?

Reply:
I tried but there was a rule in the linter when running `kcd validate` that was stating there was too much branching complexity (I cannot recall the name right now)

Reply:
this 👇 
> error  Function 'typeCharacter' has a complexity of 16. Maximum allowed is 14

Reply:
Oh, lol, bummer.... We should probably do a bigger refactor of that function or re-evaluate whether the rule is catching an actual problem here... For now, let's leave this as-is.